### PR TITLE
Improve the exit code on the payment status view (Apps-1795)

### PR DESCRIPTION
**Pull request by Fabtron**
The following changes haven been applied:
* adjust brightness if the exit code is shown
* make the exit code bigger and add a background for better scanning experience (add safezone for scanning)
* move the hint to be the sub title of the section

APPS-1795

### How to test?
Integrate this branch into an app where the exit code is shown on a checkout and and do a checkout.

### Definition of Done

- [x] Issue is linked
- [x] All requirements of the issue are fulfilled
- [x] Changelog is updated
- [ ] Documentation is updated
- [x] [Self-Review](https://www.nerdwallet.com/blog/engineering/why-you-should-be-doing-self-reviews/)
- [x] Review with the Product Owner _(Release-Variante o. Minified Build)_

#### App Tests
- [ ] Minified build has been tested _(aka. Release Build)_
- [ ] Environments have been taken care of _(Production/Staging)_
- [x] Supported languages have been tested
- [x] Light-/Dark-Mode has been tested
- [ ] Edge-Cases have been tested
- [x] Android API Levels have been taken care of _(minSdk?)_

#### Testing
- [ ] Tests have been written _(aka Unit-Tests, Integration-Tests, ...)_

### CODE REVIEW

File: ui/src/main/java/io/snabble/sdk/ui/scanner/BarcodeView.java [hunk 1]
```diff
@@ -143,6 +144,10 @@ public void setAnimateBarcode(boolean animate) {
         animateBarcode = animate;
     }
 
+    public void removeQuietZone(boolean adjust) {
```
Comment (cmaier) [inline ui/src/main/java/io/snabble/sdk/ui/scanner/BarcodeView.java:147]: Would you add a documentation for this method, please?

**Review by cmaier — CHANGES_REQUESTED** [review]
Nice :)

The same behavior should be added so _show-again_ feature:

1 | 2
--|--
Pixel_7-1733982071|Pixel_7-1733982076

**Comment by Fabtron** [comment]
> Nice :)
> 
> The same behavior should be added so _show-again_ feature:
> 1 	2
> Pixel_7-1733982071 	Pixel_7-1733982076

We have no issues there. The changes came from Support requests and only relates to the exit code in the success screen. So there will be (at this point no) no changes for this screen, since we could change it on every place where we generate a qr code via the barcode view.

File: ui/src/main/java/io/snabble/sdk/ui/scanner/BarcodeView.java (hunk 1 above)
Comment (Fabtron) [reply ui/src/main/java/io/snabble/sdk/ui/scanner/BarcodeView.java:151]: added

**Review by cmaier — APPROVED** [review]
